docs/source/delta-type-widening.md

```diff
@@ -0,0 +1,79 @@
+---
+description: Learn about type widening in Delta.
+---
+
+# Delta type widening
+
+.. note:: This feature is available in preview in <Delta> 3.2.
+
+The type widening feature allows changing the type of columns in a Delta table to a wider type. This enables manual type changes using the `ALTER TABLE CHANGE COLUMN` command and automatic type migration with schema evolution in MERGE INTO and INSERT commands.
```

Comment:
Nit: "`ALTER TABLE CHANGE COLUMN`" is quoted, but "MERGE INTO" and "INSERT" are not.